TDDProject/Program.cs

```diff
@@ -21,5 +22,7 @@
 app.UseAuthorization();
```

Comment:
Do we need this do we have test case for this ?

Reply:
we don't need for now. Hence, removing